Lmod/branches/7.0/z01-default_modules.sh

```diff
@@ -0,0 +1,11 @@
+if [ -z "$__Init_Default_Modules" -o -z "$LD_LIBRARY_PATH" ]; then
+   export __Init_Default_Modules=1;
+   if [ -f "/usr/share/modulefiles/DefaultModules.lua" ];then
+     if [ -z "$LMOD_SYSTEM_DEFAULT_MODULES" ];then
+       export LMOD_SYSTEM_DEFAULT_MODULES="DefaultModules"
```

Comment:
OK, point (2) is addressed I guess now

still need to understand how we overcome this:

```
[fgeorgatos@node001 ~]$ ll /usr/share/modulefiles/DefaultModules.lua
ls: cannot access /usr/share/modulefiles/DefaultModules.lua: No such file or directory
```